src/common/Compiler.h

```diff
@@ -262,4 +262,19 @@ inline int CountTrailingZeroes(unsigned long long x) { int i = 0; while (i < 64
 	#define UNREACHABLE()
 #endif // UNREACHABLE
 
+/* Use a C++11 braced initializer on MSVC instead of a bracket initializer
+when zeroing a struct. This works around a bug in how MSVC generates implicit
+default constructors. -- Amanieu
+
+The MSVC workaround is known to crash ICC, there is no reason to apply MSVC
+workarounds on non-MSVC compilers. -- illwieckz
+
+I believe GCC, Clang, and MSVC have all been seen to crash during compilation
+on either the () or {} forms of these constructors. -- slipher */
+#if defined(_MSC_VER)
```

Comment:
Am I blind, or are these the exact same?

Reply:
The difference is `T{};` (braces) and `T();` (parenthesis).

Reply:
Ah